### PR TITLE
Update create service tutorial

### PR DESCRIPTION
**Pull request by slowli**
This PR updates the "Create service" tutorial per exonum/cryptocurrency#39 and exonum/cryptocurrency#46 ~(the changes corresponding to the latter PR could be removed, but I've found it easier to include them right away)~.

### CODE REVIEW

**Comment by slowli** [comment]
Update to Exonum v0.5 (exonum/cryptocurrency#49) code changes are now reflected as well.

File: src/get-started/create-service.md [hunk 1]
```diff
@@ -249,15 +136,20 @@ of the wallet and produce a new instance with the modified `balance` field.
 
 ## Create Schema
 
-Schema is a structured view of the key-value storage implemented by `MemoryDB`.
-To access the storage, however, we will not use `MemoryDB` directly, but
-rather a `Fork`. `Fork` is a mutable snapshot of the database, where the changes
-can be easily rolled back; that is why it is used when dealing with transactions
-and blocks in the blockchain.
+Schema is a structured view of the key-value storage used by Exonum blockchains.
```
Comment (Maria-Nosyk) [inline src/get-started/create-service.md]: the Exonum blockchain

File: src/get-started/create-service.md [hunk 2]
```diff
@@ -268,24 +160,44 @@ a map abstraction.
...
         self.wallets().get(pub_key)
     }
 }
 ```
 
+Here, we have declared a constructor and two getter methods for the schema
+wrapping any type that allows to access it as a `Snapshot` reference
+(that is, implements the [`AsRef`][std-asref] trait from the standard library).
+`Fork` implements this trait, which means that we can construct a `CurrencySchema`
+instance above a `Fork`, and use `wallets` and `wallet` getters for it.
```
Comment (Maria-Nosyk) [inline src/get-started/create-service.md]: above the \`Fork`

File: src/get-started/create-service.md [hunk 3]
```diff
@@ -655,24 +563,125 @@ impl Service for CurrencyService {
 `CryptocurrencyApi` type implements `Api` trait of Exonum and we can use
 `Api::wire` method to connect this `Api` instance to the `Router`.
 
-## Run Service
+## Create Demo Blockchain
+
+The service is ready. You can verify that the library code compiles by running
+`cargo build` in the shell. However, we do not have a means to process requests
```
Comment (Maria-Nosyk) [inline src/get-started/create-service.md]: "we do not have a means to process requests" --> "we do not have means to process requests"

File: src/get-started/create-service.md [hunk 4]
```diff
@@ -655,24 +563,125 @@ impl Service for CurrencyService {
...
+to the service. To fix this, let us create a minimalistic blockchain network
+with one node and a single service we’ve just finished creating.
 
-We have implemented all the pieces of a minimalistic blockchain. Now,
-add `CryptocyrrencyService` to services list of the blockchain and run the demo:
+The code we are going to write is logically separate from the service itself.
+The service library could be connected to blockchains together with other services,
```
Comment (Maria-Nosyk) [inline src/get-started/create-service.md]: shouldn't it be "the blockchain" instead of "blockchains" here and above?

**Pull request merged**